test/unit/responsive/components/dropdown.test.js

```diff
@@ -1,44 +0,0 @@
-import assert from 'assert';
-import React from 'react';
-import configureMockStore from 'redux-mock-store';
-import { fireEvent } from '@testing-library/react';
-import sinon from 'sinon';
-import { renderWithProvider } from '../../../lib/render-helpers';
-import { Dropdown } from '../../../../ui/app/components/app/dropdowns/components/dropdown';
-
-describe('Dropdown components', function () {
-  const mockState = {
-    metamask: {},
-  };
-
-  const props = {
-    isOpen: true,
-    zIndex: 11,
-    onClickOutside: sinon.spy(),
-    style: {
-      position: 'absolute',
-      right: 0,
-      top: '36px',
-    },
-    innerStyle: {},
-  };
-
-  it('invokes click handler when item clicked', function () {
```

Comment:
I straight-up murdered this test cause I couldn't figure out wtf it was trying to do or why it had a mock store.

Reply:
and there was already a test file with test cases for this component in the directory in question